File: Generador/File Generado/Business/EstadoProceso.py
from DataLayer.EstadoProcesoDB import *
from EntityLayer.EstadoProcesoEntity import *


class EstadoProceso:
    def Save(Ent: EstadoProcesoSaveModel):
        try:
            return EstadoProcesoDB.Save(Ent)
        except Exception as e:
            print(e)
    
    def GetItems():
        try:
            return EstadoProcesoDB.GetItems()
        except Exception as e:
            print(e)
    
    def GetItem(Id: int):
        try:
            return EstadoProcesoDB.GetItem(Id)
        except Exception as e:
            print(e)
    
    def Delete(Id: int):
        try:
            return EstadoProcesoDB.Delete(Id)
        except Exception as e:
            print(e)
    

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.BusinessLayer/EstadoProceso.cs
using LogisticStorage.DataLayer;
using LogisticStorage.EntityLayer;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.BusinessLayer
{
    public class EstadoProceso
    {
        public static List<EstadoProcesoEntity> ObtenerItems()
        {
            EstadoProcesoDB DB = new EstadoProcesoDB();
            return DB.ObtenerItems();   
        }

    }
}
